spec/support/validate_atom_feed.rb

```diff
@@ -0,0 +1,9 @@
+require "xml/libxml"
+
+module ValidateJsonFeed
```

Comment:
`ValidateJsonFeed` 😂

Reply:
Yeahh I just noticed that ("hey why didn't I need to add `include ValidateAtomFeed`?") and decided to just fold them all together into `ValidateFeed` 😬